metadata.rb

```diff
@@ -7,7 +7,7 @@
 version          '5.0.1'
 depends          'apt', '>= 2.0'
 depends          'yum', '>= 3.0'
-depends          'lvm', '>= 1.5.1'
+depends          'lvm', '< 2.0.0'
```

Comment:
Can you lock this to `1.6.1` instead?
There is changes in that version are required to make things work correctly.